rpm/sailfish-setup.spec

```diff
@@ -24,6 +24,7 @@ if ! getent passwd sailfish-mdm >/dev/null ; then
     useradd -r -g sailfish-mdm -G privileged -d / -s /sbin/nologin sailfish-mdm || :
 fi
 groupadd -rf sailfish-radio || :
+usermod -a -G sailfish-radio nemo || :
```

Comment:
Wouldn't that give nemo the right to call privileged functions in ofono D-Bus interface? Or is the plan to make sailfish-radio a non-privileged group? What's the purpose of having this group in the first place then?

Reply:
Are there some interfaces that shouldn't be available for nemo? This change is related to https://git.merproject.org/mer-core/commhistory-daemon/merge_requests/30/diffs.

Reply:
Yes, things like `org.ofono.MessageManager.SendMessage`  and `org.ofono.VoiceCallManager.Dial` for example are disallowed for non-privileged processes. And `sailfish-radio` group is kind of considered privileged too, although I was against that.

Reply:
Pushed a new version where a new group is used for the MMS user service.

Reply:
So the plan is that `sailfish-radio` group controls access to ofono, `sailfish-mms` controls access to mms-engine, and commhistoryd has **both** `sailfish-mms` and `sailfish-radio` euids at the same time, right?